### ISSUE TITLE
brunch watch --server doesn't start http server

### ISSUE THREAD

**Issue by samelwitt** [brunch/brunch#1622]
This is my first time using Brunch instead of my usual Gulp setup, so pardon me if I'm missing something obvious.

I'm using the Dead Simple skeleton:
package.json
```
{
  "name": "brunch-app",
  "description": "Description",
  "author": "Your Name",
  "version": "0.1.0",
  "repository": {
    "type": "git",
    "url": ""
  },
  "scripts": {
    "start": "brunch watch --server",
    "build": "brunch build --production"
  },
  "dependencies": {},
  "devDependencies": {
    "auto-reload-brunch": "^2.0.0",
    "brunch": "^2.0.0",
    "clean-css-brunch": "^2.0.0",
    "css-brunch": "^2.0.0",
    "javascript-brunch": "^2.0.0",
    "uglify-js-brunch": "^2.0.0"
  }
}
```

brunch-config.js
```
exports.files = {
  javascripts: {joinTo: 'app.js'},
  stylesheets: {joinTo: 'app.css'},
  templates: {joinTo: 'app.js'}
};
```

output

```
Sites $  cd brunch/
brunch $  npm start

> brunch-app@0.1.0 start /Users/xxxx/Sites/brunch
> brunch watch --server

13 Jan 10:17:34 - info: application started on http://localhost:3333/
13 Jan 10:17:34 - info: compiled 2 files into app.js, copied 2 in 838 ms
```

### Expected behavior

I point my browser to http://localhost:3333/ and expect it to connect and GET index.html

### Actual behavior

http://localhost:3333/ never connects and eventually times out.

Just as a test, I ran a BrowserSync instance from the same directory, also using port 3333. The server started and worked as expected. Then I tried using browser-sync-brunch instead of auto-reload-brunch.

output
```
brunch $  npm start

> brunch-app@0.1.0 start /Users/xxx/Sites/brunch
> brunch watch --server

13 Jan 10:37:46 - info: application started on http://localhost:3333/
13 Jan 10:37:46 - info: compiled 2 files into app.js, copied 2 in 930 ms
[BS] Reloading Browsers...
```

But I get the same result, no http server is actually found by the browser at localhost:3333 

### Environment

```
$  npm -v
3.10.10
$  node -v
v6.9.4
$  which npm
/usr/local/bin/npm
$  which node
/usr/local/bin/node
$  brunch -v
2.8.2
$  which brunch
/usr/local/bin/brunch
```
macOS 10.12.2 (16C68)

Thank you!

**Comment by beznosd** [comment]
Hi @samelwitt 
HTTP listener on port 3333 mapped to your target folder
`public` is the default target folder

Also i would recommend you to update your brunch to the latest version, which is `2.9.1` for now

Here you can find server options to configure it in brunch-config.js
http://brunch.io/docs/config#-server-

If this will not help you, can you provide a test repo to let me reproduce your problem?

**Comment by samelwitt** [comment]
Hi @beznosd 

> HTTP listener on port 3333 mapped to your target folder
> public is the default target folder

Yes, but the issue is that there is no HTTP listener. The browser can't connect at all to localhost:3333 regardless of the rest of the path, ie /public or whatever.

I did first try 2.9.1 and downgraded to see if that fixed it, sorry I should have said that in my original post.

I could post a repo, but it's exactly the Dead Simple skeleton: https://github.com/brunch/dead-simple

I've made no changes whatsoever. I would assume it's environmental w my system, but I can't spot any likely suspects.

Thanks!
